sopel/modules/xkcd.py

```diff
@@ -123,14 +125,14 @@ def numbered_result(bot, query, latest, commanded=True):
 
 
 def say_result(bot, result, commanded=True):
-    parts = []
+    parts = [
+        result['title'],
+        'Alt-text: %s' % result['alt'],
```

Comment:
Just as a note, string concatenation is generally faster than substitution.

Not hugely important for a low-volume thing like this, but still worth keeping in mind! This is another thing that I never would have brought up if this block wasn't already getting touched.

```suggestion
        'Alt-text: ' + result['alt'],
```